src/main/java/de/tum/in/www1/artemis/config/RequestMappingConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package de.tum.in.www1.artemis.config;
+
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_BUILDAGENT;
+import static de.tum.in.www1.artemis.config.Constants.PROFILE_CORE;
+
+import java.util.List;
+
+import org.springframework.boot.autoconfigure.web.servlet.WebMvcRegistrations;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Profile;
+import org.springframework.web.servlet.mvc.method.annotation.RequestMappingHandlerMapping;
+
+import de.tum.in.www1.artemis.versioning.VersionRequestMappingHandlerMapping;
+
+/**
+ * Overwrites the default RequestMappingHandlerMapping with a versioned one.
+ */
+@Configuration
+@Profile(PROFILE_CORE + " | " + PROFILE_BUILDAGENT)
```

Comment:
Does a build agent even offer any API? Why do we need this for build agents as well?

Reply:
I want to avoid any issues. Maybe they do, maybe they don't. But I want this to be activated for all endpoints. If you don't have any, fine.
But build agents are the only profiles that get activated separately from core profiles.